R/package.r

```diff
@@ -91,6 +91,7 @@ read_meta <- function(path) {
     yaml <- list()
   } else {
     yaml <- yaml::yaml.load_file(path)
```

Comment:
This would be simpler as `yaml <- yaml::yaml.load_file(path) %||% list()`

Reply:
done